ext/quick_reference.asciidoc

```diff
@@ -169,4 +169,8 @@
 | Create Command Queues with Different Throttle Policies
 | Extension
 
+| <<cl_khr_extended_versioning,cl_khr_extended_versioning>>
+| Extend versioning of platform, devices, extensions, etc.
+| Extension
```

Comment:
Please keep this table in alphabetical order.

Reply:
Hadn't spotted the ordering. Done.